libs/jsonschema/testdata/schema-load-int/schema-invalid-enum.json

```diff
@@ -0,0 +1,10 @@
+{
+    "type": "object",
```

Comment:
Curious: why type in this one but not the other one?

Reply:
No specified reason. Does not really matter because this is not validated by the JSON schema validator.

Reply:
Automatically? :D